app/extensions/safe/webviewPreload.js

```diff
@@ -185,11 +185,6 @@ export const setupPreloadedSafeAuthApis = ( store ) =>
 
                 callbackArgs = [theCall.response];
 
-                if ( theCall.isListener )
```

Comment:
we dont have any listeners anymore?

Reply:
@joshuef right, so prior I was always receiving `null` for `appList` and `auth-web-app` would crash if an app were either revoked from the `appList` for authorised to be added to the `appList`. It was just good timing before that `auth-web-app` was still functioning because `getAuthorisedApps` would be called after the `appList` listener updated the respective state with `null`.

Reply:
fair enough 👍